.github/workflows/visual-regression-tests.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-    paths-ignore: &ignore-list
+    paths-ignore: # Until anchors are support by GitHub Actions, we have to write this list twice.
```

Comment:
Nitpick, should it be `supported` instead of `support`?

Reply:
Yes, yes it should 🤦

Reply:
Thanks @injms :+1: